app/features/build/views/build.erb

```diff
@@ -81,6 +81,8 @@
   Back to Project
 </a>
 
+<% unless build_complete %>
```

Comment:
For builds that are already complete, we don't build up a socket connection any more 👍